components/blocks/TwoColumnImageRight.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <section class="my-32">
+    <div class="base-wrapper">
+      <div>
+        <BaseHeading size="h7" class="mt-7">
+          {{ block.section_title }}
+        </BaseHeading>
+
+        <BaseHeading size="h2">
+          {{ block.heading_text }}
+        </BaseHeading>
```

Comment:
This first heading should just be a `p` tag for accessibility. Setting the `size` also sets which tag `h1, h2, etc` is used and heading elements should only be used sequentially.